pysparkle/pysparkle/data_quality/data_quality.py

```diff
@@ -0,0 +1,28 @@
+from pyspark.sql import SparkSession
+
+from pysparkle.adls_utils import set_env, set_spark_properties
+from pysparkle.data_quality.data_quality_utils import create_datasource_config, create_expectation_suite, \
+execute_validations
+
+def data_quality_main(DQ_conf):
```

Comment:
Can you update the argument name to `dq_conf`? (Python naming convention)

Reply:
This is going to change to be conf path here but I'll change it elsewhere no problem